### PR TITLE
New Copy & Run button styles implementation

### LINKED ISSUE #561: Implement the new copy & run buttons
From: https://github.com/canjs/bit-docs-html-canjs/issues/551#issuecomment-511482975

Two rough mockups::

example-expand

example-hover-copy-bar

Changes:

- [ ] The current “Run in your browser” button at the bottom of demos moves to the upper right
- [ ] The style for those action buttons (copy & run) change
- [ ] Currently the action buttons only appear on hover; with [@media (hover: hover)](https://github.com/canjs/bit-docs-html-canjs/blob/d3717becc8126037e818f71c43aeb62cc14dd348/static/toc.less#L28), we should make the action buttons always appear on non-hover devices and appear on hover on devices that support hover
- [ ] The code snippets on the homepage should also change to match this new style

Specs (you can ask @jamiemccue for any more details):

- Font size: .8em
- Font color: #858585
- Hover color: #4078c0

Screen Shot 2019-07-15 at 10 51 33 AM

### PR DESCRIPTION
**Pull request by cherifGsoul**
Fixes #561 
Depends on https://github.com/bit-docs/bit-docs-html-codepen-link/pull/14

Screen Shot 2019-08-13 at 5 22 05 PM
Screen Shot 2019-08-13 at 5 22 12 PM
Screen Shot 2019-08-13 at 5 22 16 PM
Screen Shot 2019-08-13 at 5 22 25 PM
Screen Shot 2019-08-13 at 5 22 38 PM
Screen Shot 2019-08-13 at 5 22 52 PM

### CODE REVIEW

File: static/prism.less [hunk 1]
```diff
@@ -0,0 +1,41 @@
+div.code-toolbar {
+	line-height: 0;
+	> .toolbar {
+		line-height: 0;
+		top: 0 !important;
+		right: 0 !important;
+		border-left: 1px #dfdfdf solid;
+    	border-bottom: 1px #dfdfdf solid;
```
Comment (phillipskevin) [inline static/prism.less]: Can you fix this indenting?

Comment (cherifGsoul) [reply static/prism.less]: Fixed, thanks!

File: static/canjs.less [hunk 2]
```diff
@@ -22,3 +22,4 @@
 @import "screenshots.less";
 @import "loading-bar.less";
 @import "search.less";
+@import "prism.less";
```
Comment (chasenlehara) [inline static/canjs.less]: How was this being loaded before?

File: static/prism.less [hunk 3]
```diff
@@ -0,0 +1,41 @@
+div.code-toolbar {
+	line-height: 0;
+	> .toolbar {
+		line-height: 0;
+		top: 0 !important;
+		right: 0 !important;
```
Comment (chasenlehara) [inline static/prism.less]: Are the `!important` necessary? If yes right now, can we use a more specific selector to override the other styles, or are there other styles that should be removed?

File: static/prism.less [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+.toolbar a, 
+.toolbar button, 
+.toolbar span {
+	background: #eee !important;
+	background: rgba(238, 238, 238, 1) !important;
+	color: #858585 !important;
+	padding: .35em 2.25em !important;
+	box-shadow: none !important;
+	border-radius: 0 !important;
```
Comment (chasenlehara) [inline static/prism.less]: Same thing for the `!important` in the rest of this file. We should avoid them if at all possible.

**Comment by cherifGsoul** [comment]
> This doesn’t look quite right (buttons over the expand bar):

What is not right, the style or Run button is missed?

> I don’t see the Run button integrated into the toolbar. Should that be in this PR?

@chasenlehara did you update `bit-docs-html-codepen-link` to major? https://github.com/canjs/bit-docs-html-canjs/blob/5543af358184a36f2257c77f7ced9d1d9d6990b2/make-example.js#L31

File: static/prism.less (hunk 3 above)
Comment (cherifGsoul) [reply static/prism.less]: Fixed, thanks

**Comment by cherifGsoul** [comment]
@chasenlehara I updated the styles, for notes, buttons toolbar is a sibling to the collapsible bar parent.
I updated the `make-example.js` with the last major of `bit-docs-html-codepen-link`

**Pull request merged**